usersurvey

shmcgavin opened the pull request:
@francopestilli @soichih  -- for now, this will need to live in the documentation but I wanted to put this content on a webpage we could link to on Twitter and in the newsletter.

soichih commented:
I think the file name should not contain space (I think it would work??) but I'd rather not have them.

francopestilli commented on docs/User Survey:
```diff
@@ -0,0 +1,13 @@
+
+<h2> Take the brainlife.io user survey </h2>
+
+We are inviting all registered brainlife.io users take a survey about their experience with the platform. The survey is anonymous, unless users choose to share their contact information.
```
"users take a survey"=>"users <to> take a survey"

francopestilli commented on docs/User Survey:
```diff
@@ -0,0 +1,13 @@
+
+We are asking users to participate in this survey because understanding our users’ needs and experiences with brainlife.io will help our team improve the platform to meet the needs of all students and researchers who want to use it.
+
+This survey will run until July 1 and take approximately 5 to 10 minutes to complete. Before you begin, you will read a Study Information Sheet about this study. 
```
This survey will take approximately 5 to 10 minutes to complete and run until July 1.

francopestilli commented on docs/User Survey:
```diff
@@ -0,0 +1,13 @@
+
+This survey will run until July 1 and take approximately 5 to 10 minutes to complete. Before you begin, you will read a Study Information Sheet about this study. 
+
+You can take the short survey here: [Click here to begin the survey](https://iu.co1.qualtrics.com/jfe/form/SV_0jozJ2vl9OkJSbb)
```
remove "short"

soichih commented:
@shmcgavin Will you be able to rename / and make other changes @francopestilli suggested? Do you know how to amend a PR?

Also, I can go ahead and create a page under brainlife.io website - instead of adding this under /docs

shmcgavin commented:
> @shmcgavin Will you be able to rename / and make other changes @francopestilli suggested? Do you know how to amend a PR?
> 
> Also, I can go ahead and create a page under brainlife.io website - instead of adding this under /docs

sorry, I thought @francopestilli's changes automatically changed the doc and I thought I changed the file name when I changed the title. Do I make the changes to the file name and his edits in my file? I can't figure out how to edit my file now -- do i convert the PR to a draft @soichih to be able to do that?
